Majasdarbs/03-majasdarbs/majasdarbs 3/style.css

```diff
@@ -0,0 +1,54 @@
+h1 {
+
+  text-shadow: -1px 0 black, 0 1px black, 1px 0 black, 0 -1px black;
+  font-family: 'Inter', sans-serif;
+  
+}
+h2 {
+  text-shadow: -1px 0 black, 0 1px black, 1px 0 black, 0 -1px black;
+  font-family: 'Inter', sans-serif;
+}
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+body {
+  font-family: sans-serif;
+  font-size: 16px;
+}
+nav {
+  background: rgb(126, 104, 104);
+  padding: 0 100px;
+}
+a {
+  color: white;
+  text-decoration: none;
+}
+
+@media(min-width:100px){
```

Comment:
check common breakpoints. This doesn't really do anything, since we actually check design from 320px